.ci/ci.yml.tmpl

```diff
@@ -435,16 +435,12 @@ jobs:
     - name: merge-prs
       plan:
           - get: mm-approved-prs
-            params:
-              fetch_merge: true
```

Comment:
what was fetch_merge for before? I looked it up but I'm not familiar enough with concourse or git that it made sense immediately